lib/models/crafter.rb

```diff
@@ -32,7 +33,23 @@ def self.create(user)
   end
 
   def add_office(office)
-    self.office = office
-    save
+    User.update(office => office)
+  end
+
+  def self.foreman_for_office(office)
+    User.first(:office => office, :is_foreman => true)
+  end
+
+  def self.foreman?(slack_id)
```

Comment:
Shouldn't this one filter by an office?

Reply:
I don't know, I just copied the method over from 'Apprentice`. If it does need an office, then I recommend making the change in a separate PR. The scope of this PR is just moving stuff from apprentice to user.